Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = ["crates/*"]
+exclude = ["examples/demo_trycmd"]
```

Comment:
I believe you can avoid the exclude if you put an empty `[workspace]` inside the crate